pkgs/kde/third-party/wallpaper-engine-plugin/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  extra-cmake-modules,
+  fetchFromGitHub,
+  full,
+  kpackage,
+  libplasma,
+  lib,
+  lz4,
+  mkKdeDerivation,
+  mpv,
+  pkg-config,
+  python3,
+}:
+mkKdeDerivation {
+  pname = "wallpaper-engine-kde-plugin";
+  version = "qt6";
+
+  src = fetchFromGitHub {
+    owner = "catsout";
+    repo = "wallpaper-engine-kde-plugin";
+    rev = "1e604105c586c7938c5b2c19e3dc8677b2eb4bb4";
+    hash = "sha256-bKGQxyS8gUi+37lODLVHphMoQwLKZt/hpSjR5MN+5GA=";
+    fetchSubmodules = true;
```

Comment:
Why not use system dependencies? https://github.com/catsout/wallpaper-engine-kde-plugin/blob/main/.gitmodules